<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsPage</class>
 <widget class="QWidget" name="SettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>462</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="backButton">
     <property name="styleSheet">
      <string>
       QPushButton {
           background-color: #e0e0e0;
           border: 1px solid #c0c0c0;
           border-radius: 3px;
           padding: 5px 15px;
       }
       QPushButton:hover {
           background-color: #d0d0d0;
       }
      </string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="styleSheet">
      <string>
       QLabel {
           font-size: 18px;
           color: #cc0000;
           padding: 10px;
           font-weight: bold;
       }
      </string>
     </property>
     <property name="text">
      <string>Settings</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="settingsGroup">
     <property name="styleSheet">
      <string>
       QGroupBox {
           background-color: #f0f0f0;
           border: 1px solid #c0c0c0;
           border-radius: 4px;
           padding: 10px;
       }
       QCheckBox {
           color: #333333;
           padding: 5px;
       }
       QSlider::groove:horizontal {
           background: #c0c0c0;
           height: 8px;
           border-radius: 4px;
       }
       QSlider::handle:horizontal {
           background: #cc0000;
           width: 16px;
           margin: -4px 0;
           border-radius: 8px;
       }
      </string>
     </property>
     <layout class="QVBoxLayout" name="settingsLayout">
      <item>
       <widget class="QCheckBox" name="autoUpdateCheck">
        <property name="text">
         <string>Enable Automatic Updates</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="updateFreqLabel">
          <property name="text">
           <string>Update Frequency:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="updateFreqCombo">
          <item>
           <property name="text">
            <string>Daily</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Weekly</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Monthly</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="startupScanCheck">
        <property name="text">
         <string>Scan on System Startup</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="realtimeProtectionCheck">
        <property name="text">
         <string>Enable Realtime Protection</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="scanArchivesCheck">
        <property name="text">
         <string>Scan Archive Files (zip, rar, etc.)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="notificationsCheck">
        <property name="text">
         <string>Show Notifications</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="scanSpeedLabel">
        <property name="text">
         <string>Scan Speed:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="scanSpeedSlider">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="resetDefaultsButton">
       <property name="styleSheet">
        <string>
         QPushButton {
             background-color: #666666;
             color: white;
             border: none;
             padding: 8px 16px;
             border-radius: 4px;
         }
         QPushButton:hover {
             background-color: #555555;
         }
        </string>
       </property>
       <property name="text">
        <string>Reset to Defaults</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="styleSheet">
        <string>
         QPushButton {
             background-color: #cc0000;
             color: white;
             border: none;
             padding: 8px 16px;
             border-radius: 4px;
         }
         QPushButton:hover {
             background-color: #b30000;
         }
        </string>
       </property>
       <property name="text">
        <string>Save Settings</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
